app/assets/stylesheets/frontend/views/_worldwide-organisations.scss

```diff
@@ -215,3 +141,12 @@
     }
   }
 }
+
+.contact-section {
+  border-top: 1px solid $govuk-border-colour;
```

Comment:
[`<hr class="govuk-section-break govuk-section-break--visible">`](https://design-system.service.gov.uk/styles/typography/) seems more apt here, if at all actually, this line is pretty random on this page. No action required, just a note.

Reply:
I went for the boarder because this is the convention we normally follow on pages like the collections example in the PR description. However I think you make a good point that this feels like more suitable markup. I've made a note to start a thread on the govuk-frontenders channel at the very least to get input on this.